src/kspace.cpp

```diff
@@ -33,15 +33,16 @@ static constexpr double SMALL = 0.00001;
 
 /* ---------------------------------------------------------------------- */
 
-KSpace::KSpace(LAMMPS *lmp) :
-    Pointers(lmp), eatom(nullptr), vatom(nullptr), gcons(nullptr), dgcons(nullptr)
+KSpace::KSpace(LAMMPS *lmp) : 
+    Pointers(lmp), eatom(nullptr), vatom(nullptr), gcons(nullptr), dgcons(nullptr),
+    force_poly_coeff(nullptr), energy_poly_coeff(nullptr), Fourier_poly_coeff(nullptr), Fourier_spreading_coeff(nullptr)
```

Comment:
Please keep a consistent naming scheme of the class members and make the "Fourier" lower case.